packages/widget-message-meet/src/containers/message-meet-widget/index.js

```diff
@@ -104,56 +104,85 @@ export class MessageMeetWidget extends Component {
     /* eslint-disable operator-linebreak */
     /* eslint-disable-reason: Giant list of comparisons very difficult to read and diff */
     return nextProps.conversation.activities !== props.conversation.activities
+      || nextProps.activity.get(`inFlightActivities`) !== props.activity.get(`inFlightActivities`)
```

Comment:
Do we need to do specific testing here or can we just do `nextProps.activity !== props.activity`? This would handle the status check below as well.